Content/Adding Qoutes to HomePage

AbedShowgan opened the pull request:
Added a QouteBox that contains various qoutes about the genocide to the homepage

AbedShowgan committed:
adding qoutes to main page

AbedShowgan committed:
added qoutes to homepage

AbedShowgan committed:
added qoutes to homepage

abaza738 reviewed (changes requested):
All looks good!
Couple of improvements I have in mind:
1. Could you please rename the component names to `QuoteBox` instead of `QouteBox`?
2. For the `QuoteBoxCarousel` component, to make it more reusable, let it take the quote list via props instead of hard-cording it inside the component. That way, we could use it elsewhere if we ever needed it.

AbedShowgan committed:
added json file with quotes, dynamically load the quotes

AbedShowgan commented:
> All looks good! Couple of improvements I have in mind:
> 
> 1. Could you please rename the component names to `QuoteBox` instead of `QouteBox`?
> 2. For the `QuoteBoxCarousel` component, to make it more reusable, let it take the quote list via props instead of hard-cording it inside the component. That way, we could use it elsewhere if we ever needed it.

Thanks, you made some good points!
I added a file named **"homepage_quotes.json"** in the public directory under the root project, it contains the quotes for the homepage, which are then dynamically loaded to the **index.vue** file (homepage).

I have also modified the **QuoteBoxCarousel** to use props, If any changes are needed please let me know. 

Also thanks for pointing out the misspelling of "Quotes" to "Qoutes"

AbedShowgan committed:
removed default quote placeholder

AbedShowgan committed:
removed default quote placeholder

AbedShowgan committed:
added some margin above author in quotes

abaza738 committed:
chore: fix code style
